solutions/instances/main.tf

```diff
@@ -130,6 +151,31 @@ module "observability_instance" {
   cloud_monitoring_service_endpoints = var.cloud_monitoring_service_endpoints
   enable_platform_metrics            = var.enable_platform_metrics
 
+  # IBM Cloud Logs
+  cloud_logs_provision         = var.cloud_logs_provision
+  cloud_logs_region            = var.cloud_logs_region
+  cloud_logs_instance_name     = var.prefix != null ? "${var.prefix}-cloud-logs" : var.cloud_logs_instance_name
+  cloud_logs_plan              = var.cloud_logs_plan
+  cloud_logs_tags              = var.cloud_logs_tags
+  cloud_logs_service_endpoints = var.cloud_logs_service_endpoints
+  cloud_logs_retention_period  = var.cloud_logs_retention_period
+  cloud_logs_data_storage = var.cloud_logs_provision ? {
+    logs_data = {
+      enabled         = var.enable_cloud_logs_data
+      bucket_crn      = var.enable_cloud_logs_data ? var.existing_cloud_logs_data_bucket_crn != null ? var.existing_cloud_logs_data_bucket_crn : module.cos_bucket[0].buckets[local.cloud_log_data_bucket].bucket_crn : null
+      bucket_endpoint = var.enable_cloud_logs_data ? var.existing_cloud_logs_data_bucket_endpoint != null ? var.existing_cloud_logs_data_bucket_endpoint : module.cos_bucket[0].buckets[local.cloud_log_data_bucket].s3_endpoint_direct : null
+    },
+    metrics_data = {
+      enabled         = var.enable_cloud_logs_metrics
+      bucket_crn      = var.enable_cloud_logs_metrics ? var.existing_cloud_logs_metric_bucket_crn != null ? var.existing_cloud_logs_metric_bucket_crn : module.cos_bucket[0].buckets[local.cloud_log_metric_bucket].bucket_crn : null
+      bucket_endpoint = var.enable_cloud_logs_metrics ? var.existing_cloud_logs_metric_bucket_endpoint != null ? var.existing_cloud_logs_metric_bucket_endpoint : module.cos_bucket[0].buckets[local.cloud_log_metric_bucket].s3_endpoint_direct : null
+    }
+  } : null
+  cloud_logs_existing_en_instances = var.enable_en_integration ? [{
+    en_instance_id = var.existing_en_instance_crn != null ? local.existing_en_guid : module.event_notification[0].guid
+    en_region      = var.existing_en_instance_crn != null ? local.en_region : var.en_region
+  }] : []
+
```

Comment:
Referred this code - https://github.com/terraform-ibm-modules/terraform-ibm-observability-instances/blob/main/modules/cloud_logs/main.tf#L7 to review:
* resource_group_id is missing
* left a comment for the name enable_en_integration below.
* I think...we should not create EN instance in this DA, checking more on this. Accordingly code will change.

Reply:
1. `resource_group_id` is at line 133
2. Updated the name of the variable
3. Removed the module `event_notification` to create EN instance and updated the condition accordingly.